### ISSUE TITLE
dynamic setData not working

### ISSUE THREAD

**Issue by 1mr3yn** [fengyuanchen/cropper#16]
I have a list of images, and the function looks like this:

 $('.image_list').bind('click', function() {
        $('.cropper-container img').attr('src', '{new image}');
       $(".cropper").cropper("setData", {
                                           x1: $("#dataX").val(), 
                                           y1: $("#dataY").val(),
                                          width:  $("#dataWidth").val(), 
                                          height: $("#dataHeight").val()})
                                        }); 

and the html: img class="cropper" src="{current_image}">

basically what i want is to apply the data{x1,y1,width,height} settings of the previous image into the newly added/changed image, but it seems everytime new images is loaded the data{x1,y1,width,height} reset into its default setting.

Thanks,
